novu-workflows/react/translation/app/novu/emails/welcome-email.tsx

```diff
@@ -0,0 +1,162 @@
+import {
+  Body,
+  Container,
+  Head,
+  Html,
+  Preview,
+  Section,
+  Text,
+  Link,
+  Img,
+  Row,
+  Column,
+  render,
+} from '@react-email/components';
+import * as React from 'react';
+
+const baseUrl = process.env.IMAGE_BASE_URL;
+
+export const TwitchResetPasswordEmail = ({
+  subject,
+  body,
+  linkText,
+  body2,
+}: {
+  subject: string;
+  body: string;
+  linkText: string;
+  body2: string;
+}) => {
+  return (
+    <Html>
+      <Head />
+      <Preview>{subject}</Preview>
+      <Body style={main}>
+        <Container style={container}>
+          <Section style={logo}>
+            <Img width={114} src={`${baseUrl}/twitch-logo.png`} />
+          </Section>
+          <Section style={sectionsBorders}>
+            <Row>
+              <Column style={sectionBorder} />
+              <Column style={sectionCenter} />
+              <Column style={sectionBorder} />
+            </Row>
+          </Section>
+          <Section style={content}>
+            <Text style={paragraph}>{body}</Text>
+            <Section style={center}>
+              <Link href='https://www.twitch.tv' style={link}>
+                {linkText}
+              </Link>
+            </Section>
+            <Text style={paragraph}>{body2}</Text>
+          </Section>
+        </Container>
+        <Section style={footer}>
+          <Row>
+            <Column align='right' style={{ width: '50%', paddingRight: '8px' }}>
+              <Img src={`${baseUrl}/twitch-icon-twitter.png`} />
+            </Column>
+            <Column align='left' style={{ width: '50%', paddingLeft: '8px' }}>
+              <Img src={`${baseUrl}/twitch-icon-facebook.png`} />
+            </Column>
+          </Row>
+          <Row>
+            <Text style={{ textAlign: 'center', color: '#706a7b' }}>
+              © 2022 Twitch, All Rights Reserved <br />
```

Comment:
I think we should resolve the `2022` data at runtime with `new Date().getFullYear()` to ensure it's always up to date